sample/src/main/res/values/strings.xml

```diff
@@ -0,0 +1,9 @@
+<resources>
```

Comment:
Consider the additional  supported languages, or might this could be a configurable from where will be used.

Reply:
Should be configurable from where will be used, because can't handle the labels here, depends on what you want to show, different states of what we going to use.